README.md

```diff
@@ -60,50 +60,75 @@ How would you use  `counter`, `recorder`, `gauge` and `timer` in you application
     }
 ```
 
-To ensure performance, `Metrics.makeXxx` can return a cached copy of the metric object so can be called on the hot path.
-
 ## Detailed design
 
 ### Implementing a metrics backend (eg prometheus client library)
 
-As seen above, the general function `Metrics.makeXxx` provides a metric object. This raises the question of what metrics backend I will actually get when calling `Metrics.makeXxx`? The answer is that it's configurable _per application_. The application sets up the metrics backend it wishes the whole application to use. Libraries should never change the metrics implementation as that is something owned by the application. Configuring the metrics backend is straightforward:
+As seen above, the constructors `Counter`, `Timer`, `Recorder` and `Gauge` provides a metric object. This raises the question of what metrics backend I will actually get when calling these constructors? The answer is that it's configurable _per application_. The application sets up the metrics backend it wishes the whole application to use. Libraries should never change the metrics implementation as that is something owned by the application. Configuring the metrics backend is straightforward:
```

Comment:
provides -> provide their respective metric objects

Reply:
Nit: normally trying to avoid "it's" and short forms in proposals... feel free to ignore though, more a matter of personal style I guess :)